README.md

```diff
@@ -2,32 +2,32 @@
 
 ## Introduction
 
-A module that tracks views and downloads of Islandora items, using the Drupal database.
+A module that tracks views and downloads of Islandora items, as well as solr searches, in the Drupal database.
 
 Features include:
 
-* Toggle to ignore common bots, with a configurable regex bot filter
-* View count uses session variables, and defaults to a 5 minute cooldown for repeated requests
-* IP Exclusion list to prevent artificially inflating counts while testing/developing/administrating
-* Logs all views and datastream downloads
+* ability to ignore common bots, with a configurable regex bot filter
+* ability to ignore repeated requests based on session variables, with a default cooldown period of 5 minutes
+* ability to ignore requests from specific IP addresses for testing/developing/administration
 * Views integration
 
-Note:
+Islandora Usage Stats provides the back-end framework and gathers usage data, which can be exposed through custom Views or additional modules. It also provides some built-in tools for exposing usage information.
+
+Caution:
 
 * This module, and the views/blocks it generates, does **not** respect XACML or namespace restrictions.
-* As this is a server-side tracking solution, a caching layer could prevent accesses from being recorded.  If this is impacting you a [solution](https://github.com/discoverygarden/islandora_ga_reports) using JavaScript may work better.
+* As this is a server-side tracking solution, a caching layer can prevent usage from being accurately recorded.  If this is impacting you a [solution using JavaScript](https://github.com/discoverygarden/islandora_ga_reports) may work better.
```

Comment:
If we're gonna plug Google Analytics as a Usage Stats option, can we plug Matomo as well? https://github.com/DiegoPino/islandora_piwik/